### ISSUE TITLE
Fix npm publishing workflow

### ISSUE THREAD

**Issue by kibertoad** [kibertoad/better-fd-slicer#6]
https://github.com/kibertoad/better-fd-slicer/actions/runs/7725563295

I assume we need to configure a token

**Comment by H4ad** [comment]
Yes ahhaah I think we can rebase and remove those commits/releases/tags.

**Comment by H4ad** [comment]
Do you want to release it in your account? If so, just create a simple NPM Token for CI.
